client/store/cart.js

```diff
@@ -57,8 +57,16 @@ export const getFromDB = () => async dispatch => {
 
 export const createOrder = () => async (dispatch, getState) => {
   const total = getState().total;
-  await axios.post('/api/orders', { amount: parseFloat(total) });
+  const cart = getState().cart;
+  const order = await axios.post('/api/orders', { amount: parseFloat(total) });
+  const orderId = order.data.id;
+  cart.forEach(function(item) {
+    item.orderId = orderId;
+  });
+  console.log(cart);
+  await axios.put('/api/carts', cart);
   dispatch(createdOrder());
+  dispatch(clearCart());
```

Comment:
when an order is created, add an "orderId" key to cart items on state, then do a PUT request for those cart instances